ui/src/tool-bar/workspace-dialog.tsx

```diff
@@ -78,6 +81,8 @@ export const WorkspaceDialog: React.FC<WorkspaceDialogProps> = ({
         .map((e) => t(`workspace-dialog.${e}`))
         .concat(designNameNotUnique ? [t('workspace-dialog.name-not-unique')] : []);
 
+    React.useEffect(() => nameInputRef?.current?.focus());
```

Comment:
Tässä dialogissahan on muitakin kenttiä, ei kai nimeä haluta fokusoida joka renderöintikerralla?

Reply:
Totta, tämähän tosiaan ajautuu näin joka renderillä. Tarkoitus oli lisätä tälle tyhjät depsut, sillä nimikenttä halutaan focusata vain dialogin avauksessa. Korjattu nyt